src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -92,9 +93,21 @@ export function isNative(appName: string): boolean {
 export async function stop(appName: string): Promise<void> {
   const gameInfo = getGameInfo(appName)
   const {
-    install: { executable = undefined }
+    install: { executable = undefined, platform = undefined }
   } = gameInfo
 
+  if (platform && platform === 'web') {
```

Comment:
Do you think this approach is better than passing the abortController to the open game browser window command like we are doing on Heroic and I pushed to the overlay branch?

On that approach, we send the kill command with the exact appName and then we do not need to iterate through all windows and match the URL, because HP already knows which window belongs to each browser game.

Reply:
yeah that's better. just fixed